doc/classes/DirAccess.xml

```diff
@@ -104,6 +104,17 @@
 				[b]Note:[/b] This method is implemented on macOS, Linux, and Windows.
 			</description>
 		</method>
+		<method name="create_temp" qualifiers="static">
+			<return type="DirAccess" />
+			<param index="0" name="prefix" type="String" default="&quot;&quot;" />
+			<param index="1" name="keep" type="bool" default="false" />
+			<description>
+				Creates a temporary directory.
+				If [param prefix] is not empty, the directory name will be preceded by the [String] you passed, separated by a [code]-[/code].
+				If [param keep] is [code]true[/code], the directory will outlive the [DirAccess] instance. When [code]false[/code], the directory will be deleted, including all its files.
+				If the returned value is [code]null[/code], use [method get_open_error] to get some context.
```

Comment:
Although I'm not a fan of this wording, it would be consistent with `open`'s description
```suggestion
				Returns [code]null[/code] if opening the directory failed. You can use [method get_open_error] to check the error that occurred.
```